kornia/enhance/jpeg.py

```diff
@@ -0,0 +1,564 @@
+from typing import Optional, Tuple
+
+import torch
+
+from kornia.color import rgb_to_ycbcr, ycbcr_to_rgb
+from kornia.core import Device, Dtype, Module, Tensor
+from kornia.core.check import (
+    KORNIA_CHECK,
+    KORNIA_CHECK_IS_TENSOR,
+    KORNIA_CHECK_SHAPE,
+)
+from kornia.geometry import rescale
+
+QUANTIZATION_TABLE_Y: Tensor = torch.tensor(
```

Comment:
move these tensors to be lazyloaded by putting them into a function. Otherwise, these tensors will be created when importing occurs; the function can have device and dtype as parameter, to initialize these tensor in the right place instead of moving/casting it after

Reply:
Not sure if correct but implemented like this:

https://github.com/kornia/kornia/blob/046443331956d8703683944e384cf21c75fb5d07/kornia/enhance/jpeg.py#L17-L32

Reply:
yeah, perfect